tools/phenomenal/w4m/isa2mzml.xml

```diff
@@ -0,0 +1,69 @@
+<!-- vi: se fdm=marker : -->
+<tool id="isa2mzml" name="ISA to mzML" version="1.1.0">
+
+	<description>Extract mzML files from an ISA dataset and output a collection of mzML dataset.</description>
+
+	<!-- Command {{{1 -->
+
+	<command><![CDATA[
+		## @@@BEGIN_CHEETAH@@@
+
+		extract-from-isa
+		-i "$isa.extra_files_path"
```

Comment:
I presume `extract-from-isa` will create a directory called `mzML` in the CWD.

Reply:
Yes, that's what it does. And if I understand well, after that the files are copied into individual datasets by Galaxy and then form a collection with them.